docs/clientsidequeries.md

```diff
@@ -0,0 +1,85 @@
+---
+id: clientqueries
+title: Client-side Queries
+---
+
+Additionaly graphback also allows users to generate client-side queries based on the data model.
+These can be used by client side frameworks like [uniform.tools](https://uniforms.tools) to create
+GraphQL client applications.
+
+Graphback asks whether you want to generate client-side queries in the `graphback init` command. Answering it yes,
+will set `client` in `config.json` to `true`.
+
+For a data model having
+```
+type Note {
+  id: ID!
+  title: String!
+  description: String!
+}
+```
+running `graphback generate` will generate all operations which are enabled in `config.json` or using `directives`.
+
+## Examples
+### Fragment
+```
+import gql from "graphql-tag"
+
+export const NoteFragment = gql`
```

Comment:
It is important that we done it this way.

Uniforms can base on NoteFragments to build forms input. Separate tooling I have build will apply common patterns - for example hide id field and generate labels based on the metadata provided in directives